cosmic-settings/src/pages/sound.rs

```diff
@@ -17,8 +86,8 @@ impl page::Page<crate::pages::Message> for Page {
         Some(vec![
             sections.insert(output()),
             sections.insert(input()),
-            sections.insert(alerts()),
-            sections.insert(applications()),
+            // sections.insert(alerts()),
```

Comment:
Do you want to have these commented out, or just remove them?

Reply:
I'll remove them